### ISSUE TITLE
Docs search is broken

### ISSUE THREAD

**Issue by nikitavoloboev** [vitest-dev/vitest#1420]
### Describe the bug

<img width="611" alt="image" src="https://user-images.githubusercontent.com/6391776/171894265-a4edf66e-187c-49fa-afce-f4aa15d86c8a.png">

Searching anything will not return any results for me. Using Safari on macOS.

### Reproduction

Search for anything using the search bar.

### System Info

```shell
System:
    OS: macOS 12.4
    CPU: (10) arm64 Apple M1 Pro
    Memory: 113.36 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 17.9.0 - ~/Library/Caches/fnm_multishells/34445_1654161919462/bin/node
    Yarn: 1.22.15 - /opt/homebrew/bin/yarn
    npm: 8.5.5 - ~/Library/Caches/fnm_multishells/34445_1654161919462/bin/npm
  Browsers:
    Chrome Canary: 104.0.5099.0
    Safari: 15.5
```
```


### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by userquin** [comment]
@nikitavoloboev it seems the `searchParameters` not working, removing that entry works on local, with it stops working (maybe we need to configure something on algolia):
```ts
      searchParameters: {
        facetFilters: ['tags:en'],
      },
```

![imagen](https://user-images.githubusercontent.com/6311119/171907911-78c6b71d-b35d-403a-8ef3-d0f3a3a5be23.png)

**Comment by userquin** [comment]
fixed: [6082b69](https://github.com/vitest-dev/vitest/commit/6082b6942d7aff35b17fa3952715b4a4a2bddfbf)
